docs/source/api.rst

```diff
@@ -15,6 +15,21 @@ Constructor
 
         This function returns a new Docxtemplater Object
 
+.. code-block:: text
+
+    new Docxtemplater(zip, options)
```

Comment:
new Docxtemplater(zip[ ,options]) 

Because options is optional

Reply:
Worked on it and now the options object is optional.